docs/help.md

```diff
@@ -8,6 +9,28 @@
 * [Log levels as labels instead of numbers](#level-string)
 * [Pino with `debug`](#debug)
 
+<a id="exit-logging"></a>
+## Exit logging
+
+When a Node process crashes from uncaught exception, exits due to a signal, 
+or exits of it's own accord we may want to write some final logs – particularly
+in cases of error. 
+
+Writing to a Node.js stream on exit is not necessarily guaranteed, and naively writing
+to an Extreme Mode logger on exit will definitely lead to lost logs. 
+
+To write logs in an exit handler, create the handler with `pino.final`:
+
+```js
+process.on('uncaughtException', pino.final(logger, (err, finalLogger) => {
+  finalLogger.error(err)
```

Comment:
can you add a `process.exit(1)` here? Also an example with `'unhandledRejection'` would be nice.